DMTN-222.tex

```diff
@@ -259,6 +268,7 @@ \subsection{Distribution}
 Exporting the ticket-level CHAINED collection will export all of the children CALIBRATION collections, making it the preferred way to export calibrations.
 It is strongly recommended that the file permissions are checked in the \verb|export_directory| to ensure that the files are world readable.
 The import process will create files with the same permissions on some repositories, so this check is essential to prevent processing problems.
+Note that the known failures where exported calibrations had incorrect permissions have all been fixed.
```

Comment:
I'd rather remove any instructions to `chmod` if that's not going to be a problem anymore.